Audacia.Log/Audacia.Log.csproj

```diff
@@ -3,26 +3,28 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <Authors>Audacia</Authors>
-    <Description>Standardized SeriLog logging configuration for Audacia projects.</Description>
-    <Copyright>Copyright © Audacia 2020</Copyright>
+    <Description>Standardized logging configuration for Audacia projects.</Description>
+    <Copyright>Copyright © Audacia $([System.DateTime]::UtcNow.Year)</Copyright>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
     <PackageIcon>package-icon.png</PackageIcon>
-    <Version>1.1.0</Version>
+    <Version>1.1.1</Version>
```

Comment:
This can stay at 1.1.0, the pipeline will bump it

Reply:
Reverted.